media/js/base/mozilla-notification-banner.js

```diff
@@ -16,6 +16,7 @@ if (typeof Mozilla === 'undefined') {
     NotificationBanner.COOKIE_CODE_ID = 'mozilla-notification-banner';
     NotificationBanner.COOKIE_EXPIRATION_DAYS = 21; // default cookie expiry 21 days
     NotificationBanner.COOKIE_INTERACTION_VALUE = 'interacted';
+    NotificationBanner.SAMPLE_RATE = 0.05; // 5% sample rate limit
```

Comment:
This is just a default value, it can easily be overridden per instance if required. I don't think we have any plans to display more than one notification on the same page at the same time (at least, I really hope not from a UX perspective), and we don't have any requirements to have different sample rates per config either, so until we do i think this is probably acceptable. Let me know if you think differently though?